src/styles/blocks/card.scss

```diff
@@ -0,0 +1,150 @@
+:root {
+  --blue-accent: #00acdc;
+  --white: #fff;
+  --gray-color: #616070;
+  --dark-blue-color: #060b35;
+}
+
+.page {
```

Comment:
it looks like a separate bem block, you should create a separate file for each bem block